code/src/java/pcgen/gui2/doomsdaybook/NameGenPanel.java

```diff
@@ -690,7 +687,7 @@ private void loadCatalogDD()
 			join.retainAll(sexes);
 			Collections.sort(join, new DataElementComperator());
 
-			Vector<RuleSet> catalogs = new Vector<RuleSet>();
+			List<RuleSet> catalogs = new ArrayList<>();
```

Comment:
On second thought, I see what you are going after, but PCGen's coding style so far has been to spell out the arguments of the generics, and until that style is changed, it should be followed.